html-forms/html-forms/html-forms-quiz-notes.md

```diff
@@ -0,0 +1,55 @@
+# html-forms-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is the purpose of an HTML `form` element?
+  The purpose of an HTML 'form' element is to represent a document section containing interactive controls for submitting information.
+
+- Give five examples of form control elements.
+  Five examples of form control elements are:
+
+  1. text
+  2. textarea
+  3. select
+  4. email
+  5. radio
+
```

Comment:
Some of these are correct and some of these aren't. `email` for example is a value for the `type` attribute and not an actual element itself. Think of actual element names like `label`